### PR TITLE
add one GET test

### PR DESCRIPTION
**Pull request by MikeTrek**

### CODE REVIEW

File: src/main/java/routes/Get.java [hunk 1]
```diff
@@ -14,4 +15,8 @@ public Get(RequestSpecification requestSpecification) {
     public Response getEmployees() {
         return this.request.get(route);
     }
+
+    public Response getSingleEmployees() {
```
Comment (wizzardio) [inline src/main/java/routes/Get.java:19]: better to use `getEmployee`.

File: src/main/java/routes/Get.java [hunk 2]
```diff
@@ -6,6 +6,7 @@
 public class Get {
     private RequestSpecification request;
     public String route = "/employees";
+    public String route1 = "/employee/1";
```
Comment (wizzardio) [inline src/main/java/routes/Get.java:9]: It's not comfortable to use hard-coded values in URL. 
Better to refactor this one
```
public String employeesRoute = "/employees";
public String employeeRoute = "/employee/";
```

After it in method, you can take an additional parameter (ID). And modify String to use this param.
For example (code may be a little bit incorrect)
```
getEmployee(int id) {
    String route = employeeRoute + id;
```

File: src/test/java/get/GetTest.java [hunk 3]
```diff
@@ -14,12 +14,24 @@
     @BeforeTest
     public void beforeTest() {
         getRouter = new Get(this.requestSpecification);
+
```
Comment (wizzardio) [inline src/test/java/get/GetTest.java:17]: redundant line here

File: src/test/java/get/GetTest.java [hunk 4]
```diff
@@ -14,12 +14,24 @@
...
     }
+
+    @Test
+    public void getSingleEmployeeReturns200() {
+        requestSpecification.given().log().all();
+        Response response = getRouter.getSingleEmployees();
+        int statusCode = response.getStatusCode();
+        Assert.assertEquals(statusCode, 200);
+    }
+
```
Comment (wizzardio) [inline src/test/java/get/GetTest.java:35]: redundant line here

**Review by wizzardio — COMMENTED** [review]
Looks good. Just a few comments to fix.

**Comment by MikeTrek** [comment]
fixed

**Pull request merged**